packages/libs/react-ui/src/styles/sprinkles.css.ts

```diff
@@ -103,6 +103,10 @@ const responsiveProperties = defineProperties({
       'min-content': 'min-content',
       'max-content': 'max-content',
     },
+    maxHeight: {
+      '100%': '100%',
+      maxContent: 'auto',
```

Comment:
What do we want this to be?

Reply:
I think this would be a good question to ask in the channel. We likely don't need to give all the "size" variables here